dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsAPIImpl.java

```diff
@@ -473,23 +473,71 @@ public Experiment start(String experimentId, User user)
         DotPreconditions.checkState(persistedExperiment.goals().isPresent(), "The Experiment needs to "
                 + "have the Goal set.");
 
+        final List<Experiment> runningExperimentsOnPage = list(ExperimentFilter.builder()
+                .pageId(persistedExperiment.pageId())
+                .statuses(Set.of(RUNNING))
+                .build(), user);
+
+        if(!runningExperimentsOnPage.isEmpty()) {
+            if(persistedExperiment.scheduling().isEmpty()) {
```

Comment:
This is ok Vico. If it doesn't have Scheduling it means it is meant to be run now, which should not be possible because there are running Expeirments already. I'm gonna make it a variable to give it more meaning.

Reply:
Cool